packages/design-system/src/components/Button/Button.md

```diff
@@ -54,6 +54,20 @@ Use as a final confirmation for a destructive action such as deleting. These are
 </ComponentHtmlMarkup>
 ```
 
+### Text Button
+
```

Comment:
Idea: could you add text button as a separate subsection? Like a ButtonGroup, it will be more visible.

Reply:
Maybe we should create sub-sections for each type of button? @kamilmateusiak wdyt?

Reply:
@kamilmateusiak ☝️ 🙏

Reply:
I am against subsections for each type of button. It's exactly the same component with different configuration

Reply:
I'll try to figure out something here.

Reply:
Let's keep it as it is. I put my suggestion to our jira task.